src/components/data/EntitiesContent.js

```diff
@@ -385,11 +386,17 @@ const EntitiesContent = ({
         h(MenuButton, {
           onClick: () => setAddingColumn(true)
         }, ['Add column']),
+        h(MenuDivider),
```

Comment:
UX requested a divider to separate actions that operate on the table as a whole from actions that operate on selected rows.

Reply:
Posted [my thoughts](https://broadinstitute.slack.com/archives/C34GAKQ5R/p1652899589845669) in case they lead to further improvements in the future 😊